test/victoria-logs-cluster_test.go

```diff
@@ -3,46 +3,31 @@ package test
 import (
 	"context"
 	"fmt"
-	"strings"
 	"testing"
 
-	"github.com/gruntwork-io/terratest/modules/helm"
 	"github.com/gruntwork-io/terratest/modules/k8s"
-	"github.com/gruntwork-io/terratest/modules/random"
 	"github.com/stretchr/testify/require"
 	appsv1 "k8s.io/api/apps/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/wait"
 )
 
-// TestVictoriaLogsClusterInstallDefault tests that the victoria-logs-cluster chart can be installed with default values.
+// TestVictoriaLogsClusterInstallDefault tests that the %s chart can be installed with default values.
 func TestVictoriaLogsClusterInstallDefault(t *testing.T) {
-	const helmChartPath = "../charts/victoria-logs-cluster"
-
-	namespaceName := fmt.Sprintf("vlogcluster-%s", strings.ToLower(random.UniqueId()))
-	k8sOpts := k8s.NewKubectlOptions("", "", namespaceName)
-
-	helmOpts := &helm.Options{
-		BuildDependencies: true,
-		KubectlOptions:    k8sOpts,
-		ExtraArgs: map[string][]string{
-			"upgrade": {"--create-namespace", "--wait"},
-		},
-	}
-
-	// Install the chart and verify no errors occurred.
-	releaseName := fmt.Sprintf("vlogcluster-%s", strings.ToLower(random.UniqueId()))
-	defer helmCleanup(context.Background(), t, k8sOpts, helmOpts, releaseName)
-	helm.Upgrade(t, helmOpts, helmChartPath, releaseName)
-
-	k8sClient, err := k8s.GetKubernetesClientFromOptionsE(t, k8sOpts)
-	require.NoError(t, err)
+	t.Parallel()
```

Comment:
Does this chart use CRDs? If yes we can't use Parallel here as it may clash with other charts installing CRDs like vmoperator and k8s-stack